keyboards/newgame40/newgame40.h

```diff
@@ -0,0 +1,19 @@
+#ifndef NEWGAME40_H
+#define NEWGAME40_H
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
If you're using `LAYOUTS = ortho_4x10`, then you should use that here: 

```suggestion
#define LAYOUT_ortho_4x10( \
```